src/components/molecules/Visualizer/Plugin/types.ts

```diff
@@ -1,4 +1,4 @@
-import { Clock } from "cesium";
+import { Clock } from "../Engine/ref";
```

Comment:
Define Clock type here and import it in `Engine/Cesium/ref.ts` etc. since this file should be a master with all types explicitly defined for the plugin's API.

Current: Plugin/types ---use--> Engine/ref (Clock type)
Ideal: Plugin/types (Clock type) <---use-- Engine/ref

Reply:
I fixed in [refactor: define Clock type in Plugin/types](https://github.com/reearth/reearth-web/pull/285/commits/7675054cdf2aeb16fcb794fd50b3fc565796c201)